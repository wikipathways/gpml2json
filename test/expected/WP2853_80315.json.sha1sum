8c67c5f3817ee747327b62a5ff2b700f2ba98d66 ?-
