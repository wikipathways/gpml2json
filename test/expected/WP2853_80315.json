{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c87a4","b8ed8","f31e9"],"drawAs":"rect","gpmlElementName":"Pathway","height":444.7005444646098,"kaavioType":"Group","name":"Endoderm Differentiation","padding":5,"type":["Pathway"],"width":445.50545698542817,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","comments":[{"source":"WikiPathways-description","content":"Model depicting endoderm specification based on the literature and highly enriched gene expression profiles via comparison across dozens of independent induced and embryonic pluripotent stem cell lines, following differentiation to multiple lineages (ectoderm, mesoderm, endoderm, embryoid body). The underlying genomic data can be obtained from: \r\nhttps://www.synapse.org/#!Synapse:syn1773109"}],"organism":"Homo sapiens","standardName":"Endoderm Differentiation","displayName":"Endoderm Differentiation","textContent":"Name: Endoderm Differentiation\nOrganism: Homo sapiens","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=Endoderm%20Differentiation&species=Homo%20sapiens&doSearch=1","id":"hash://sha256/405678f6be1238474eb19252497dcbdcf082c386c57f0e1d530b0bc5ea52d429"},"entityMap":{"b8ed8":{"textContent":"GeneProduct","id":"b8ed8","wpType":"GeneProduct","x":89,"y":250,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c87a4":{"id":"c87a4","cellularComponent":"Cell","borderStyle":"double","x":35.93633405835186,"y":144.01719135499872,"width":346.68889280190217,"height":272.18335310961106,"zIndex":-2,"backgroundColor":"#ccffcc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#c0c0c0","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Cell"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f31e9":{"textContent":"PAX9","id":"f31e9","x":355.50545698542817,"y":279.46243741125534,"width":60,"height":20,"zIndex":33898,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Ensembl","dbId":"ENSG00000198807","wpType":"Unknown","type":["DataNode","SingleFreeNode","Unknown"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"}}}
